jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarProcessorTest.java

```diff
@@ -281,4 +282,46 @@ public void testExplodeMode_standard_withoutClassPathInManifest_containsOnlyClas
             AbsoluteUnixPath.get("/app/explodedJar/class1.class"),
             AbsoluteUnixPath.get("/app/explodedJar/class2.class"));
   }
+
+  @Test
+  public void testExplodeMode_standard_computeEntrypoint_allLayersPresent()
+      throws IOException, URISyntaxException {
+    Path standardJar =
+        Paths.get(Resources.getResource(STANDARD_JAR_WITH_CLASS_PATH_MANIFEST).toURI());
+    Path destDir = temporaryFolder.newFolder().toPath();
```

Comment:
Just so you know, if you don't need a nested directory or multiple directories, you can use the root: `temporaryFolder.getRoot().toPath()`.